### PR TITLE
Add LZ4 support

### PR DESCRIPTION
**Pull request by jfilak**

### CODE REVIEW

File: tests/compress.at [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+TS_RETURN_MAIN
+]])
+])
+
+
+## --- ##
+## LZ4 ##
+## --- ##
+
+AT_TESTFUN_COMPRESS([lz4])
```
Comment (mhabrnal) [inline tests/compress.at]: Macro is named AT_TESTFUN_DECOMPRESS, see line 5.

File: tests/compress.at [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+
+## -- ##
+## XZ ##
+## -- ##
+
+AT_TESTFUN_COMPRESS([xz])
```
Comment (mhabrnal) [inline tests/compress.at]: the same here

**Comment by mhabrnal** [comment]
Otherwise looks good.

**Comment by mhabrnal** [comment]
Pushed:
https://github.com/abrt/libreport/commit/78951f065b0ec116bc60c8488f8784b708971daf
https://github.com/abrt/libreport/commit/aedc53a7900670bad2c5cbd8f7f2312806097be1
https://github.com/abrt/libreport/commit/414d0762df912662f988967ca6ba7d9ec1cea970
https://github.com/abrt/libreport/commit/7fd06174d0ee454a0fa4e35f489ede013c32a7db

**Comment by ernestask** [comment]
> Otherwise looks good.

Aha, except for the commented-out `PKG_CHECK_MODULES` lines.

**Pull request closed without merging**